examples/oft-composer-library/README.md

```diff
@@ -0,0 +1,162 @@
+<p align="center">
+  <a href="https://layerzero.network">
+    <img alt="LayerZero" style="width: 400px" src="https://docs.layerzero.network/img/LayerZero_Logo_White.svg"/>
+  </a>
+</p>
+
+<p align="center">
+  <a href="https://layerzero.network" style="color: #a77dff">Homepage</a> | <a href="https://docs.layerzero.network/" style="color: #a77dff">Docs</a> | <a href="https://layerzero.network/developers" style="color: #a77dff">Developers</a>
+</p>
+
+<h1 align="center">LayerZero OFT Composer Library</h1>
+
+<p align="center">
+  <a href="https://docs.layerzero.network/v2/developers/evm/oft/quickstart" style="color: #a77dff">Quickstart</a> | <a href="https://docs.layerzero.network/contracts/oapp-configuration" style="color: #a77dff">Configuration</a> | <a href="https://docs.layerzero.network/contracts/options" style="color: #a77dff">Message Execution Options</a> | <a href="https://docs.layerzero.network/v2/developers/evm/composer/overview" style="color: #a77dff">Composer Overview</a>
+</p>
+
+<p align="center">
+  A Composer library to integrate LayerZero composer contracts with the Omnichain Fungible Token (OFT) standard.
+</p>
+
+- [What is an Omnichain Fungible Token?](#what-is-an-omnichain-fungible-token)
+- [Using Composer with OFTs](#using-composer-with-ofts)
+- [LayerZero Hardhat Helper Tasks](#layerzero-hardhat-helper-tasks)
+- [Developing & Deploying Contracts](#developing-contracts)
+- [Connecting Contracts](#connecting-contracts)
+
+## What is an Omnichain Fungible Token?
+
+The Omnichain Fungible Token (OFT) standard extends the ERC20 interface to enable seamless cross-chain token transfers without the need for asset wrapping or intermediary chains. By combining LayerZero’s OApp Contract Standard with ERC20’s `_burn` and `_mint` methods, OFTs maintain a unified token supply across multiple blockchains.
```

Comment:
we should just link to https://docs.layerzero.network/v2/concepts/glossary#oft-omnichain-fungible-token